tests/acceptance/test_veb_negative.py

```diff
@@ -224,7 +224,10 @@ def test_handle_consensus_report_data_second_exit(contract, ref_slot):
     no_global_index = (module_id, no_id) = (1, 33)
     validator_id = 1
     validator_key = contracts.node_operators_registry.getSigningKey(no_id, validator_id)[0]
-    validator = LidoValidator(validator_id, validator_key)
+
+    # set validator index to the next one to avoid NodeOpValidatorIndexMustIncrease error
+    last_requested_validator_index = contract.getLastRequestedValidatorIndices(module_id, [no_id])[0]
+    validator = LidoValidator(index=last_requested_validator_index + 1, pubkey=validator_key)
```

Comment:
It looks inconsistent, last requested validator index and the first validator key.